### PR TITLE
improvement and fixes

### PR DESCRIPTION
**Pull request by AlwinEsch**
To make add-on better usable and implement missing settings features.

### CODE REVIEW

**Comment by AlwinEsch** [comment]
Note about commit "[move libfec parts to own folder](https://github.com/kodi-pvr/pvr.rtlradio/pull/11/commits/0190b7ff6a471f5b901fbe1c273007c4a40dddda)", after this comes much times crash on playbacks.
As in code no differences to before and only on other binary place in add-on lib, it seems thats somewhere a mem write overflow where destroy some mem data in processes.

File: pvr.rtlradio/resources/language/resource.language.en_gb/strings.po [hunk 1]
```diff
@@ -126,7 +132,10 @@ msgstr ""
 
 # 30202 can be reused
 # 30203 can be reused
-# 30204 can be reused
+
+msgctxt "#30204"
+msgid "Enable coarse corrector (for receivers with >1kHz error)."
```
Comment (ksooo) [inline pvr.rtlradio/resources/language/resource.language.en_gb/strings.po]: Could you add a hint for end users when to set this to which of the three values? What is the ">1kHz error"?

File: src/addon.cpp [hunk 2]
```diff
@@ -1627,6 +1633,34 @@ ADDON_STATUS addon::SetSetting(std::string const& settingName,
...
+    bool bvalue = settingValue.GetBoolean();
+    if (bvalue != m_settings.dabradio_coarse_corrector)
+    {
+
+      m_settings.dabradio_coarse_corrector = bvalue;
+      log_info(__func__, ": setting dabradio_coarse_corrector changed to ", bvalue);
+    }
+  }
+
+  // dabradio_coarse_corrector
```
Comment (ksooo) [inline src/addon.cpp]: This should read `// dabradio_coarse_corrector_type` (copy/paste error I guess)

File: src/pvrtypes.h [hunk 3]
```diff
@@ -159,6 +159,16 @@ struct settings
   // Specifies the output gain for the DAB DSP
   float dabradio_output_gain;
 
+  // dabradio_output_gain
```
Comment (ksooo) [inline src/pvrtypes.h]: This comment seems not to fit here (copy/paste from above?)

**Comment by AlwinEsch** [comment]
Thanks @ksooo for review, commented places are fixed.

About the settings I increased the text and now shown as this, what you think, OK as this or another idea about?:
Screenshot 2024-04-01 025326

**Comment by ksooo** [comment]
Yes, this is better. Thanks.

**Review by ksooo — APPROVED** [review]

**Pull request merged**